test/harness/wasm-module-builder.js

```diff
@@ -100,17 +107,23 @@ class WasmFunctionBuilder {
 
   addBody(body) {
     for (let b of body) {
-      if (typeof b !== 'number' || (b & (~0xFF)) !== 0 )
```

Comment:
It seems that in this case, the version in the spec repo was updated to do even stricter tests. Hence we should merge this change back to v8.